### ISSUE TITLE
Manage Tags text changes - 

### ISSUE THREAD

**Issue by StuMackenzie** [MotusDev/Motus-TO-DO#787]
Please replace current text with below. @leberrigan Take note of language for other areas where text is being updated.

Few things that need to be cleaned up.

- it's not 'our' system.

- This sentence is confusing and misleading:

"You must register a deployment for each tag prior to deployment. This is separate from tag registration. You can set a default deployment date during registration. You can set a default deployment date during registration."

I would prefer that no dates are set prior to deployment - it provides an incentive NOT to input metadata.

Also the use of register for two different meanings. 

See changes below.

##### Main Page####

The table below lists all the devices that are registered to your project. Click on one of the rows to see more information about your tag, including a table of deployments.

**Tags must be registered prior to deployment.** Tag registration and deployment information is how Motus knows what tags to look for at any given time, and the precise information gathered from recordings cannot be made without the physical tag in the hand. How to Register Lotek Tags (link)

CTT tags should be registered automatically after purchasing, but please confirm that your tags are listed before deployment. 

If you need tags moved to another project, please contact us. Do not attempt to register tags to our system twice.

Lotek NanoTags: - _why is this specific to Lotek tags?_

**The Motus server will not search for tags that are not deployed**. You should enter deployment information for each tag prior to deployment. You can set a default deployment date during registration for Lotek Tags. For CTT tags you'll have to enter a deployment date.

If you do not deploy your tags at the expected time and are storing them for later use, you should change your deployments to this later date or delete the deployments entirely.

Once your tags have been deployed on an animal, update your deployment information as soon as possible. This can be done in bulk by clicking the 'Upload Tag Deployments' button below. Accurate and up to date metadata is critical to ensuring you have the most up to date data, and to support optimal operation of the network as a whole and make the greatest use of the data. 

#####When you click on any tag#####
***Perhaps all of this information should be combined with information on the main page as well.***
    
- Tag deployments tell the Motus serve when your tags are active. Without deployments, the tag finder algorithm doesn't know when to look for your tag so you will have no detections. You can have as many deployments for a tag as you want, they just can't overlap.
-     You must enter deployment information prior to the actual deployment of your tags. The Motus server only searches for registered and deployed tags, and they cannot be easily searched for retroactively. You may never get your tag detections if you do not have deployment information .
-     Deployments should be ended if and only if the tag has been removed from the animal and/or the tag is deactivated at any point.
-     Remaining tag life is calculated using accumulated deployment time. We use the tag lifespan to tell the tag finder algorithm when to search for your tag. This means that if you start your deployments too early, or if you don't end them when you should (see above), tag finder might think your tag has run out battery life too early and stop will searching for it. Of course, this only applies to battery-powered tags.
-     Providing accurate deployment information ensures that the tag detection can be assigned to the correct animal. It also ensures data provided to the public and other collaborators is accurate. Please do not enter a species for tag testing and mark the tag as a test in the deployment.
-     A deployment created cannot be destroyed, for now. Please ensure that you enter your information carefully. The ability to modify or delete deployments will be added in the future. You can also contact us (link) for any problems.

**Comment by leberrigan** [comment]
Related to #775 

Text has been updated.

### Main page now reads:

> <p>The table below lists all the devices that are registered to your project. </b>Click on one of the rows to see more information about your tag, including a table of deployments. </p>
> <p><b>Tags must be registered prior to deployment.</b> Tag registration and deployment information is how Motus knows what tags to look for at any given time, and the precise information gathered from recordings cannot be made without the physical tag in the hand. <a href="https://motus.org/tag-registration/" target='_blank' rel='noopener'>How to Register Lotek Tags</a></p>
> <p>CTT tags should be registered automatically after purchasing, but please confirm that your tags are listed before deployment. </p>
> <p>If you need tags moved to another project, please <a href='https://motus.org/contact'>contact us</a>. Do not attempt to register tags to our system twice. </p>
> 
> <p><b>The Motus server will not search for tags that are not deployed.</b> You need to enter deployment information for each tag prior to deployment. You can set a default deployment date <a href="https://archived.sensorgnome.org/VHF_Tag_Registration/#section_5" target='_blank' rel='noopener'>during registration for Lotek Tags</a>. For CTT tags you'll have to enter a deployment date.</p>
> <p><b>Once your tags have been deployed on an animal, update your deployment information as soon as possible.</b> This can be done in bulk by clicking the 'Upload Tag Deployments' button below. Accurate and up to date metadata is critical to ensuring you have the most up to date data, and to support optimal operation of the network as a whole and make the greatest use of the data.</p>
> <p><b>If you do not deploy your tags at the expected time and are storing them for later use,</b> you should change your deployments to this later date or delete the deployments entirely.</p>
> <p><b>Remaining tag life is calculated using accumulated deployment time.</b> We use the tag lifespan to tell the Motus server when to search for your tag. This means that if you start your deployments too early, or if you don't end them when you should (see above), the Motus server might think your tag has run out battery life too early and will stop searching for it. Of course, this only applies to battery-powered tags.</p>

### When you click on any tag: 

> 
> <ul><li><b>Tag deployments tell the Motus server when your tag was active.</b> Without deployments, the <i>tag finder</i> algorithm doesn't know when to look for your tag so you will have no detections. You can have as many deployments for a tag as you want, they just can't overlap.</li>
> <li><b>You must enter deployment information prior to the actual deployment of your tags. </b> The Motus server only searches for registered and deployed tags, and they cannot be easily searched for retroactively. You may never get your tag detections if you do not have deployment information. </li>
> <li><b>Deployments should be ended</b> if and only if the tag has been removed from the animal and/or the tag is deactivated at any point.</li>
> <li><b>Remaining tag life is calculated using accumulated deployment time.</b> We use the tag lifespan to tell the Motus server when to search for your tag. This means that if you start your deployments too early, or if you don't end them when you should (see above), the Motus server might think your tag has run out battery life too early and will stop searching for it. Of course, this only applies to battery-powered tags.</li>
> <li><b>Providing accurate deployment information ensures that the tag detection can be assigned to the correct animal.</b> It also ensures data provided to the public is accurate. Please do not enter a species for tag testing and mark the tag as a test in the deployment.</li>
> <li><b>A deployment created cannot be destroyed, for now.</b> Please ensure that you enter your information carefully. The ability to modify or delete deployments will be added in the future. You can also <a href='contact'>contact us</a> for any problems. </li></ul>

**Comment by jsayers-bsc** [comment]
I had some comments in mind for the previous git but didn't get there in time. Here are a few suggestions based on the new one. There may be a benefit to having the same information displayed on both Main Page and the individual Tag page, so this hopefully could apply to both (minus the registration bit since only registered tags are on the Tags page).

> __Tags must be registered prior to deployment.__ Tag registration and deployment information is how Motus knows what tags to look for at any given time, and the precise information gathered from recordings cannot be made without the physical tag in the hand. It is recommended you enter a default deployment date when registering Lotek tags (see below). How to Register Lotek Tags

> CTT tags should be registered automatically after purchasing, but please confirm that your tags are listed before deployment.

> If you need tags moved to another project, please contact us. Do not attempt to register tags twice. 

Can we combine the first second and third paragraphs to something like:

>__The tag finder algorithm will not search for tags that do not have an active deployment record.__ ~~You need to enter deployment information for each tag prior to deployment.~~ Once your tags have been deployed on an animal, update your deployment information as soon as possible, even if you had set a default deployment date during registration. ~~for Lotek Tags. For CTT tags you'll have to enter a deployment date. Accurate and up to date metadata is critical to ensuring you have the most up to date data, and to support optimal operation of the network as a whole and make the greatest use of the data.~~ Tag deployments can be updated in bulk by clicking the 'Upload Tag Deployments' button below. 

There have been a few people emailing about confusion regarding the need to register deployments prior to the actual deployment of the tag. We do recommend creating a default deployment date when registering Lotek tags, though that really is  in the event data from receivers that contain detections of their tag is uploaded prior to them updating their metadata, or if those researchers need their data very soon after deployment.

And because of how Remaining Tag Life affects whether or not detections of that tag will be properly identified, there is a risk of inadvertently encouraging people to set default deployments too early. 

"If only tags with deployments will be detected, and I can set a deployment early, I'm just going to set deployment dates to start tomorrow, even though I may not actually deploy these until next year, with Covid and all that." — *imaginary collaborator trying to cover their bases without knowing how remaining tag life factors into this all*

(side note: won't setting a future deployment date be a thing of the past anyway once automatic registration of Lotek tags begins?)

> __If you do not deploy your tags at the expected time and are storing them for later use__, you should change your deployments to this later date or contact Motus to have the deployments deleted entirely. This is due to how Remaining Tag Life is factored into the tag finder algorithm (see below).

[They'll have to contact us to have deployments deleted.]

> __Remaining Tag Life is an estimate of how long a tag will continue to transmit.__ It is based on 1) the estimated battery life provided by the manufacture for that tag model, burst interval, and duty cycle—plus an additional buffer—as well as 2) the start date of your deployment. Once the Remaining Tag Life has elapsed, the deployment is considered to be ended and the tag finder algorithm will stop looking for this particular tag.

> Because most tags will never be recovered and the true deployment end date known, the Remaining Tag Life is a crucial factor in determining whether or not your tags will be searched for. This means that if you start your deployments too early, or if you don't end them when you should (see above), the tag finder algorithm might think your tag has run out battery life too early and will stop searching for it. Of course, this only applies to battery-powered tags.

I'm not sure if the above is too wordy, but Remaining Tag Life seems to be an incredibly important concept that we really don't explain very well, yet it factors into tag deployments in a very important ways that users aren't often aware of.

Also, because of the importance in having active deployments for tags to be identified, we may want to rephrase the bit on ending deployments to shift the emphasis.

> __Deployments should not ended unless the tag has been removed from the animal and/or the tag is deactivated at any point.__ If the tag is not recovered, the Remaining Tag Life will be used to determine when the battery has died and the tag no longer searched for.

> __Accurate and up-to-date metadata is critical to the success of Motus.__ It ensures you have the most up-to-date data for your project, and also ensures the highest quality data for the hundreds of other projects making use of the collaborative Motus network. 

We should also probably keep the phrasing consistent between `tag finder algorithm` and `Motus server`. In this context, "tag finder algorithm" (or tagfinder) seems more appropriate and consistent with how users conceptualize it (though perhaps there are nuances I'm missing that warrant differentiation).

Those are some suggestions that hopefully would work for both pages. Would have been better had I gotten them in to Issue 775 but better late than never?

By the way Lucas, all these improvements, updates, and additions to the documentation you've cranked out in the last couple weeks are awesome and long overdue!

**Comment by leberrigan** [comment]
Thanks for the edits! Definitely adds some clarity. I've added changes to the last paragraph of the main page to try to clear things up, but I think it could still use some fixing.

### Main page now reads:

> <p>The table below lists all the devices that are registered to your project. </b>Click on one of the rows to see more information about your tag, including a table of deployments. </p>
> <p><b>Tags must be registered prior to deployment.</b> Tag registration and deployment information is how Motus knows what tags to look for at any given time, and the precise information gathered from recordings cannot be made without the physical tag in the hand. It is recommended you enter a default deployment date when registering Lotek tags. <a href="https://motus.org/tag-registration/" target='_blank' rel='noopener'>How to Register Lotek Tags</a></p>
> <p>CTT tags should be registered automatically after purchasing, but please confirm that your tags are listed before deployment. </p>
> <p>If you need tags moved to another project, please <a href='https://motus.org/contact'>contact us</a>. Do not attempt to register tags twice. </p>
> <p><b>The <i>tag finder</i> algorithm will not search for tags that do not have an active deployment record</b> Once your tags have been deployed on an animal, update your deployment information as soon as possible, even if you had <a href="https://archived.sensorgnome.org/VHF_Tag_Registration/#section_5" target='_blank' rel='noopener'>set a default deployment date during registration of Lotek tags</a>.Tag deployments can be updated in bulk by clicking the 'Upload Tag Deployments' button below. Accurate and up to date metadata is critical to ensuring you have the most up to date data, and to support optimal operation of the network as a whole and make the greatest use of the data.</p>
> <p><b>If you do not deploy your tags at the expected time and are storing them for later use,</b> you should change your deployments to this later date or <a href='https://motus.org/contact'>contact Motus</a> to have the deployments deleted entirely. This is due to how Remaining Tag Life is factored into the <i>tag finder</i> algorithm (see below).</p>
> <p><b>The <i>tag finder</i> algorithm will stop searching for your tags once it thinks the battery has died.</b> Most tags are never retrieved once deployed so we have to assume a time when they run out of battery life. This is based on the calculated lifespan for that tag model provided by the manufacturer, plus a generous buffer.This means that if you start your deployments too early, or if you don't end them when you should (see above), the <i>tag finder</i> algorithm might think your tag has run out battery life too early and will stop searching for it. Of course, this only applies to battery-powered tags; <i>tag finder</i> will search for solar-powered tags indefinitely.</p>


### When you click on any tag: 

> 
> <ul><li><b>Tag deployments tell the Motus server when your tag was active.</b> Without deployments, the <i>tag finder</i> algorithm doesn't know when to look for your tag so you will have no detections. You can have as many deployments for a tag as you want, they just can't overlap.</li>
> <li><b>You must enter deployment information prior to the actual deployment of your tags. </b> The Motus server only searches for registered and deployed tags, and they cannot be easily searched for retroactively. You may never get your tag detections if you do not have deployment information. </li>
> <li><b>Deployments should not ended unless the tag has been removed from the animal and/or the tag is deactivated at any point.</b> If the tag is not recovered, the Remaining Tag Life will be used to determine when the battery has died and the tag no longer searched for.</li>
> <p><b>The <i>tag finder</i> algorithm will stop searching for your tags once it thinks the battery has died.</b> Most tags are never retrieved once deployed so we have to assume a time when they run out of battery life. This is based on the calculated lifespan for that tag model provided by the manufacturer, plus a generous buffer.This means that if you start your deployments too early, or if you don't end them when you should (see above), the <i>tag finder</i> algorithm might think your tag has run out battery life too early and will stop searching for it. Of course, this only applies to battery-powered tags; <i>tag finder</i> will search for solar-powered tags indefinitely.</p>
> <li><b>Accurate and up-to-date metadata is critical to the success of Motus.</b> It ensures you have the most up-to-date data for your project, and also ensures the highest quality data for the hundreds of other projects making use of the collaborative Motus network.</li>
> <li><b>A deployment created cannot be destroyed, for now.</b> Please ensure that you enter your information carefully. The ability to modify or delete deployments will be added in the future. You can also <a href='contact'>contact us</a> for any problems. </li></ul>

**Comment by cjardine-bsc** [comment]
RE:It is recommended you enter a default deployment date when registering Lotek tags

I suggest we call this "anticipated deployment date" instead of default. 

RE: The tag finder algorithm will not search for tags that do not have an active deployment record

This isn't really true. It will search for undeployed tags based on datebin and lifespan. Its not a big deal, but telling users this might confuse them if they get detections for undeployed tags. 

I made a few language tweaks to the below section. 

The tag finder algorithm will stop searching for your tags once the expected lifespan has passed. This is based on the calculated lifespan for that tag model provided by the manufacturer, plus a generous buffer. In the rare case that a tag is recovered after it has been deployed,  the deployment end date is used. If you start your deployments too early, or if you don't end them when you should (see above), the tag finder algorithm might think your tag has run out battery life too early and will stop searching for it. Of course, this only applies to battery-powered tags; tag finder will search for solar-powered tags indefinitely.

**Comment by denislepage** [comment]
-          tag finder will search for solar-powered tags indefinitely.

I guess we haven’t decided on a rule for when to stop looking for Lotek solar tags. I assume that there will be an upper limit where we reallow the tag code to be reissued? Perhaps based on the time of the last detection with a sufficient run length.

**Comment by denislepage** [comment]
*   You must enter deployment information prior to the actual deployment of your tags. The Motus server only searches for registered and deployed tags, and they cannot be easily searched for retroactively. You may never get your tag detections if you do not have deployment information.
Is this realistic though? Less ideal, but still useful, is to enter the metadata as soon as possible after the actual deployment. We could point out that if they know the date, they should enter it before the deployment, and come back later to make any corrections with additional details.

Perhaps something like this (a bit too wordy, perhaps):

Entering proper tag deployment information in a timeline fashion is HIGHLY IMPORTANT. Without it, your tags detections may simply not be found and available to you. The most important aspects of a tag deployment record are the tag deployment date and the species, which you should enter as soon as possible, and preferably BEFORE the actual deployment or immediately thereafter. Once your deployments are made, we encourage make any adjusting needed your deployment metadata as soon as possible. Note that tag detections that are missed due to incomplete metadata record should get picked up again later on a second processing round, but you may have to wait several weeks or months before you see your data.

**Comment by leberrigan** [comment]
### Main page now reads:

> <p>The table below lists all the devices that are registered to your project. </b>Click on one of the rows to see more information about your tag, including a table of deployments. </p>
> <p><b>Tags must be registered prior to deployment.</b> Tag registration and deployment information is how Motus knows what tags to look for at any given time, and the precise information gathered from recordings cannot be made without the physical tag in the hand. It is recommended you enter an anticipated deployment date when registering Lotek tags. <a href="https://motus.org/tag-registration/" target='_blank' rel='noopener'>How to Register Lotek Tags</a></p>
> <p>CTT tags should be registered automatically after purchasing, but please confirm that your tags are listed before deployment. </p>
> <p>If you need tags moved to another project, please <a href='https://motus.org/contact'>contact us</a>. Do not attempt to register tags twice. </p>
> <p><b>The <i>tag finder</i> algorithm will not search for tags that do not have an active deployment record</b> Once your tags have been deployed on an animal, update your deployment information as soon as possible, even if you had <a href="https://archived.sensorgnome.org/VHF_Tag_Registration/#section_5" target='_blank' rel='noopener'>set an anticipated deployment date during registration of Lotek tags</a>.Tag deployments can be updated in bulk by clicking the 'Upload Tag Deployments' button below. Accurate and up to date metadata is critical to ensuring you have the most up to date data, and to support optimal operation of the network as a whole and make the greatest use of the data.</p>
> <p><b>If you do not deploy your tags at the expected time and are storing them for later use,</b> you should change your deployments to this later date or <a href='https://motus.org/contact'>contact Motus</a> to have the deployments deleted entirely. This is due to how Remaining Tag Life is factored into the <i>tag finder</i> algorithm (see below).</p>
> <p><b>The <i>tag finder</i> algorithm will stop searching for your tags once the expected lifespan has passed.</b>  This is based on the calculated lifespan for that tag model provided by the manufacturer, plus a generous buffer. In the rare case that a tag is recovered after it has been deployed, the deployment end date is used. If you start your deployments too early, or if you don't end them when you should (see above), the <i>tag finder</i> algorithm might think your tag has run out battery life too early and will stop searching for it. Of course, this only applies to battery-powered tags; tag finder will search for solar-powered tags indefinitely.</p>


### When you click on any tag: 

> 
> <ul><li><b>Tag deployments tell the Motus server when your tag was active.</b> Without deployments, the <i>tag finder</i> algorithm doesn't know when to look for your tag and you may never get detections. You can have as many deployments for a tag as you want, they just can't overlap.</li>
> <li><b>You should enter an anticipated deployment date prior to the actual deployment of your tags. </b>This way the Motus server will start searching for your tag closer to the date you actually plan on deploying it. If no anticipated deployment date is provided at registration, the <i>tag finder</i> algorithm defaults to the beginning of the month the tags were registered.
Remember, the <i>tag finder</i> algorithm only searches for registered and deployed tags, and they cannot be easily searched for retroactively. You may never get your tag detections if you do not have deployment information. </li>
> <li><b>Deployments should not ended unless the tag has been removed from the animal and/or the tag is deactivated at any point.</b> If the tag is not recovered, the Remaining Tag Life will be used to determine when the battery has died and the tag no longer searched for.</li>
> <li><b>The <i>tag finder</i> algorithm will stop searching for your tags once the expected lifespan has passed.</b>  This is based on the calculated lifespan for that tag model provided by the manufacturer, plus a generous buffer. In the rare case that a tag is recovered after it has been deployed, the deployment end date is used. If you start your deployments too early, or if you don't end them when you should (see above), the <i>tag finder</i> algorithm might think your tag has run out battery life too early and will stop searching for it. Of course, this only applies to battery-powered tags; tag finder will search for solar-powered tags indefinitely</li>
> <li><b>Accurate and up-to-date metadata is critical to the success of Motus.</b> It ensures you have the most up-to-date data for your project, and also ensures the highest quality data for the hundreds of other projects making use of the collaborative Motus network.</li>
> <li><b>A deployment created cannot be destroyed, for now.</b> Please ensure that you enter your information carefully. The ability to modify or delete deployments will be added in the future. You can also <a href='contact'>contact us</a> for any problems. </li></ul>
